settings.lua

```diff
@@ -2494,6 +2494,14 @@ function RidgeMarkersAutomatic:isDisabled()
 	return self.vehicle.cp.driver and not AIDriverUtil.hasAIImplementWithSpecialization(self.vehicle, RidgeMarker)
 end
 
+---@class KeepCurrentSteering : BooleanSetting
+KeepCurrentSteering = CpObject(BooleanSetting)
+function KeepCurrentSteering:init(vehicle)
+	BooleanSetting.init(self, 'KeepCurrentSteering', 'COURSEPLAY_KEEP_CURRENT_STEERING',
```

Comment:
the second argument is the member variable name in the settings and should be lower camelcase like all other settings (hopefully), so keepCurrentSteering.

Reply:
Done on next commit.